system_tests/bigquery.py

```diff
@@ -33,6 +35,15 @@ def _make_dataset_name(prefix):
     return '%s%s' % (prefix, unique_resource_id())
 
 
+def _load_json_schema(filename='bigquery_test_schema.json'):
+    from google.cloud.bigquery.table import parse_schema_resource
+    where = os.path.dirname(__file__)
```

Comment:
> 1. I typically use os.path.abspath to be "defensive".

OK.  The interpreter is the thing generating it, not a user, but it can't hurt.

> 2.  Why not make a constant?

It is only used by this function.